### PR TITLE
Vue popups

### PR DESCRIPTION
**Pull request by GoodDayForSurf**

### CODE REVIEW

File: packages/vue/src/pages/crm-contact-list/crm-contact-list.vue [hunk 1]
```diff
@@ -167,6 +167,14 @@
       @close="isPanelOpen = false"
     />
   </div>
+
+  <form-popup
+    title="New Contact"
+    v-model:is-visible="isAddContactPopupOpened"
```
Comment (dxvladislavvolkov) [inline packages/vue/src/pages/crm-contact-list/crm-contact-list.vue:173]: Why do we use v-model here?

Comment (GoodDayForSurf) [reply packages/vue/src/pages/crm-contact-list/crm-contact-list.vue:173]: for able to open popup again

**Review by dxvladislavvolkov — APPROVED** [review]

File: packages/vue/src/pages/crm-contact-list/crm-contact-list.vue [hunk 2]
```diff
@@ -242,6 +257,12 @@ const customizePhoneCell = (cellInfo: {value: string}) => {
 
   return formatPhone(value.toString());
 };
+
+const onSaveUserNewForm = () => {
+  if (validationEngine.validateGroup(newUserValidationGroup).isValid) {
```
Comment (groshenkovamarina) [inline packages/vue/src/pages/crm-contact-list/crm-contact-list.vue:262]: May be newContactValidationGroup will be better?

File: packages/vue/src/utils/media-query.ts [hunk 3]
```diff
@@ -47,6 +49,11 @@ function getScreenSizeInfo(): ScreenSizeInfo {
 
 export const screenInfo = ref(getScreenSizeInfo());
 
+export const isSmallScreen = () => {
+  const screenSizes = getScreenSizeInfo();
```
[the remark this answers is not in the document]
Comment (GoodDayForSurf) [reply packages/vue/src/utils/media-query.ts:53]: yep..refactor

File: packages/vue/src/utils/media-query.ts [hunk 4]
```diff
@@ -49,11 +49,6 @@ function getScreenSizeInfo(): ScreenSizeInfo {
 
 export const screenInfo = ref(getScreenSizeInfo());
 
-export const isSmallScreen = () => {
```
Comment (dxvladislavvolkov) [inline packages/vue/src/utils/media-query.ts]: Why do you remove this?

Comment (GoodDayForSurf) [reply packages/vue/src/utils/media-query.ts]: look at refactored form-popup. I found more simpler way

**Review by groshenkovamarina — APPROVED** [review]

**Review by dxvladislavvolkov — APPROVED** [review]

**Pull request merged**